bentoml/frameworks/transformers.py

```diff
@@ -104,7 +104,7 @@ def _load_from_directory(self, path):
         tokenizer = getattr(
             import_module("transformers"), self._tokenizer_type
         ).from_pretrained(path)
-        self._model = {"model": transformers_model, "tokenizer": tokenizer}
+        return {"model": transformers_model, "tokenizer": tokenizer}
```

Comment:
changed this to actually just return model information after looking at how all the other frameworks currently set their `self._model` property